DESCRIPTION

```diff
@@ -24,6 +24,7 @@ Imports:
     fs (>= 1.5.2),
     glue (>= 1.6.2),
     haven (>= 2.5.0),
+    hms,
```

Comment:
Having looked at [the changelog](https://hms.tidyverse.org/news/index.html), I think we should require > 1.1.0 as that version had some breaking changes which might affect us.
```suggestion
    hms (>= 1.1.0),
```